README.md

```diff
@@ -71,6 +72,17 @@ A such annotated secret looks like the following:
 
 The default password for these keystores is `changeme`. The password can be changed by adding the following optional annotation: `cert-utils-operator.redhat-cop.io/java-keystore-password: <password>`. The alias of the certificate inside the keystore is `alias`.
 
+### ConfigMaps
```

Comment:
can you update the docs to reflect the new annotation available?

Reply:
added docs for annotations and defaults